Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -228,6 +246,22 @@ public void TestSimpleLargeResultSet()
             }
         }
 
+        [Test]
+        public void TestCommandPrepareThrowsNotImplemented()
+        {
+            using (IDbConnection conn = new SnowflakeDbConnection())
+            {
+                // Arrange
+                conn.ConnectionString = ConnectionString;
+                conn.Open();
+
+                // Act
+                IDbCommand cmd = conn.CreateCommand();
+
+                // Assert
+                Assert.Throws<NotImplementedException>(() => cmd.Prepare());
```

Comment:
We shouldn't do integration test to test that Prepare throws an exception. It is very good candidate for unit tests.

Reply:
Sure, I'll move it to the unit tests